### ISSUE TITLE
[Bug]: Auto(Based on display) detects the wrong aspect ratio.

### ISSUE THREAD

**Issue by MEMORIEmusic** [PlayCover/PlayCover#1113]
### Describe the bug

Upon installing a new game, the default `Auto(Based on display)` resolution setting of PlayCover will set the aspect ratio to 16:9 regardless of the display’s aspect ratio. 

This issue exists on both macOS Ventura and macOS Sonoma. 

### Steps to reproduce

1. Install a new game 
2. Put the game in full screen
3. Visible black bars at the top and the bottom, indicating aspect ratio is set to 16:9

### Expected behaviour

Aspect ratio should change in accordance with the display. 

### What version of PlayCover are you using?

Nightly/beta

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by zanderp25** [comment]
Which game is having this issue? The automatic feature works fine with Genshin Impact and Honkai: Star Rail, but unfortunately, it might be a limitation with the game itself.

**Comment by MEMORIEmusic** [comment]
Having the issue with both Genshin Impact and Honkai: Star Rail.

**Comment by MEMORIEmusic** [comment]
<img width="1800" alt="Screenshot 2023-10-01 at 10 41 20" src="https://github.com/PlayCover/PlayCover/assets/79436281/f62dc550-af4a-4b8f-a4e0-e2bd4d95b19b">

<img width="1061" alt="Screenshot 2023-10-01 at 10 41 27" src="https://github.com/PlayCover/PlayCover/assets/79436281/f7fb290d-9043-46e8-830d-190b2d0abcf1">

**Comment by zanderp25** [comment]
Do you happen to have an M2 mac? In that case, hopefully my PR #1116 should fix this issue.

**Comment by MEMORIEmusic** [comment]
> Do you happen to have an M2 mac? In that case, hopefully my PR #1116 should fix this issue.

Nope. Mine is 14” M1 Pro MacBook Pro. Tho I’ll still try the latest nightly.

**Comment by zanderp25** [comment]
> Nope. Mine is 14” M1 Pro MacBook Pro.

Weird. I have the same one and it works fine for me. 

I noticed you chose the "More Space" scaling option and changed "Automatically hide and show the menu bar" to Never. After changing to those settings, it still works fine for me. I'm using the 3.0.0 pre-release right now.

**Comment by zanderp25** [comment]
Update: I updated to 3.0.0-beta.1 and it still works fine.

<img width="1008" alt="Screenshot 2023-10-02 at 09 48 27" src="https://github.com/PlayCover/PlayCover/assets/13000207/33772ff8-151e-4444-aadf-1b7469c53c05">
<img width="1800" alt="Screenshot 2023-10-02 at 09 48 52" src="https://github.com/PlayCover/PlayCover/assets/13000207/72125d9a-aab7-48f5-a514-86a8e231b788">

**Comment by MEMORIEmusic** [comment]
> > Nope. Mine is 14” M1 Pro MacBook Pro.
> 
> Weird. I have the same one and it works fine for me.
> 
> I noticed you chose the "More Space" scaling option and changed "Automatically hide and show the menu bar" to Never. After changing to those settings, it still works fine for me. I'm using the 3.0.0 pre-release right now.

It only happens upon a fresh install of the game. If I change it to 1080p 16:10, and then back to Auto, it automatically starts launching in 16:10
